app/views/users/show.html.erb

```diff
@@ -1,12 +1,36 @@
-<h1>Users#show</h1>
-<p>Find me in app/views/users/show.html.erb</p>
-<!-- app/views/users/index.html.erb -->
-<h1>All Users</h1>
-<p>Here is a list of all users:</p>
-<!-- Implement code to display the list of users -->
-
-<!-- app/views/users/show.html.erb -->
-<h1>User Details</h1>
-<p>Here are the details for the user:</p>
-<!-- Implement code to display the details of a single user -->
-
+<h1 class='title'>Welcome to Blog-Haven</h1>
+<div class= "container">
+<% if @user %>
+    <div class='user'>
+      <div class='avatar'>
+        <%= image_tag(@user.photo) %>
+      </div>
+    <div><h2><%= @user.name %></h2></div>
+    <div><p>Number of Posts: x</p></div>
+    </div>
+    <div class='user-info'>
+      <div><h2>Bio</h2></div>
+      <div><p><%= @user.bio %><br/>Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellat aliquid eius fuga. Nulla reprehenderit, ipsa quam, asperiores itaque totam vel amet iste ducimus pariatur distinctio reiciendis quasi hic, illum sit.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Assumenda, quas magni. Iure deserunt voluptates accusantium quam maxime tempore sint veniam laudantium? Aperiam placeat dolor quasi facilis! Dicta quasi soluta fugiat cupiditate facere consequuntur ipsum! Ea beatae, adipisci delectus sunt rerum veniam velit odio sit deleniti deserunt nobis facere nostrum ex!</p>
+      </div>
+      </div>
+  
+<div class="recent-posts">
+  <% if @user.most_recent_posts.any? %>
+    <% @user.most_recent_posts.each do |post| %>
+        <div class='recent-post'>
+          <h3><%= link_to post.title, user_post_path(@user, post), class: 'post-title' %> Post</h3>
+          <p><%= post.text %><br/>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Assumenda, quas magni. Iure deserunt voluptates accusantium quam maxime tempore sint veniam laudantium? Aperiam placeat dolor quasi facilis! Dicta quasi soluta fugiat cupiditate facere consequuntur ipsum! Ea beatae, adipisci delectus sunt rerum veniam velit odio sit deleniti deserunt nobis facere nostrum ex! </p>
```

Comment:
- [ ] You do not need the dummy text, kindly remove them

Reply:
well resolved! ✔️